pyaedt/modules/Boundary.py

```diff
@@ -3350,3 +3351,738 @@ def delete(self):
+    def add_link(self, node1, node2, value, name=None):
+        """Create links in the network object.
+
+        Parameters
+        ----------
+        node1 : str or int
+            String containing one of the node name that the link is connecting or integer containing
+            the id of the face. If id is used and the node associated to the corresponding face is not
+            created yet, it is added atuomatically.
+        node2 : str or int
+            String containing one of the node name that the link is connecting or integer containing
+            the id of the face. If id is used and the node associated to the corresponding face is not
+            created yet, it is added atuomatically.
+        value : str or float
+            String containing the value and unit of the connection. If a float is passed, by default an
+            R-Link is added to the network and the ``"cel_per_w"`` unit is used.
+        name : str, optional
+            String containing the name of the link. Default is ``None`` and a name is
+            automatically generated.
+
+        Returns
+        -------
+        bool
+            ``True`` if successful. ``False`` otherwise.
+
+        Examples
+        --------
+
+        >>> import pyaedt
+        >>> app = pyaedt.Icepak()
+        >>> network = pyaedt.modules.Boundary.Network(app)
+        >>> box = app.modeler.create_box([5, 5, 5], [20, 50, 80])
+        >>> faces_ids = [face.id for face in box.faces]
+        >>> connection = {"Name": "LinkTest", "Connection": [faces_ids[1], faces_ids[0]], "Value": "1cel_per_w"}
+        >>> network.add_links_from_dictionaries(connection)
+
+        """
+        if not isinstance(value, str):
+            value = str(value) + "cel_per_w"
+        if name is None:
+            new_name = True
+            while new_name:
+                name = generate_unique_name("Link")
+                if name not in self.links.keys():
+                    new_name = False
+        new_link = self._Link(node1, node2, value, name, self)
+        self._links.append(new_link)
+        self._add_to_props(new_link, "Links")
+        return True
+
+    @pyaedt_function_handler()
+    def add_links_from_dictionaries(self, connections):
+        """Create links in the network object.
+
+        Parameters
+        ----------
+        connections : dict or list of dict
+            Dictionary or list of dictionaries containing the links between nodes. Each dictionary is
+            composed by:
+            - ``"Link"``: a 3-item list with the two node that the link is connecting and the value and
+            unit of the link. The node of the connection can be referred to with their name (str) or
+            face id (int). The link type (resistance, heat transfer coefficient, or mass flow) is
+            determined automatically from the unit.
+            Optional key:
+            - ``"Name"``: a string that prescribes the name of the link.
```

Comment:
```suggestion
        connections : dict or list of dict
            Dictionary or list of dictionaries containing the links between nodes. Each dictionary
            consists of these elements:
            
            - ``"Link"``: a three-item list consisting of the two nodes that the link is connecting and the value and
               unit of the link. The node of the connection can be referred to with the name (str) or
               face ID (int). The link type (resistance, heat transfer coefficient, or mass flow) is
               determined automatically from the unit.
            - Optional key:
            - ``"Name"``: a string specifying the name of the link.

```
I am unclear if "Optional key" is an element in the dictionary.